steward/src/cellars/cellar_v2_5.rs

```diff
@@ -260,6 +260,36 @@ pub fn get_encoded_function(call: FunctionCall, cellar_id: String) -> Result<Vec
 
             Ok(CellarV2_5Calls::DecreaseShareSupplyCap(call).encode())
         }
+        Function::SetSharePriceOracle(params) => {
+            let cellar_id_normalized = normalize_address(cellar_id.clone());
+            let oracle_in = normalize_address(params.oracle.clone());
+            let allowed_oracles = [
+                normalize_address("0x72249f0199EACf6230DEF33A31e80CF76de78f67".to_string()),
+                normalize_address("0xC47278B65443cE71CF47E8455BB343F2DB11B70e".to_string()),
+            ];
+            let allowed_registry_ids = [U256::from(3), U256::from(5)];
+            let registry_id_in = string_to_u256(params.registry_id.clone())?;
+            if !cellar_id_normalized.eq(CELLAR_TURBO_SWETH)
+                || !allowed_oracles.contains(&oracle_in)
+                || !allowed_registry_ids.contains(&registry_id_in)
```

Comment:
Technically the validation here is a mapping of either [3, 0x722...] or [5, 0xc47....], not any combination of the two.